charts/addon-manager/values.yaml

```diff
@@ -1,5 +1,5 @@
 # Image of the cluster-gateway instances
-image: yue9944882/cluster-gateway-addon-manager:v1.1.7.fix.2
+image: yue9944882/cluster-gateway-addon-manager:v1.1.7.fix.3
```

Comment:
let's use a formal oamdev docker hub repo

Reply:
will switch to oamdev repo from the next release on i.e. 1.1.8